### PR TITLE
[AIRFLOW-6838] Add subcommands to Breeze

### PR DESCRIPTION
**Pull request by potiuk**
---
Issue link: [AIRFLOW-6838](https://issues.apache.org/jira/browse/AIRFLOW-6838)

Make sure to mark the boxes below before creating PR: [x]

- [x] Description above provides context of the change
- [x] Commit message/PR title starts with `[AIRFLOW-NNNN]`. AIRFLOW-NNNN = JIRA ID<sup>*</sup>
- [x] Unit tests coverage for changes (not needed for documentation changes)
- [x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions.
- [x] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

<sup>*</sup> For document-only changes commit message can start with `[AIRFLOW-XXXX]`.

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

**Comment by potiuk** [comment]
This PR has been originated by @mik-laj - but I completed it with much better help. With this change 'breeze' introduces sub-commands instead of --switches everywhere and its interface is vastly improved. See for yourself in BREEZE.rst.

@mik-laj -> Please take a look. Kudos for pushing Breeze in this direction!

**Comment by potiuk** [comment]
@mik-laj (others) I am wondering if we should not also split flags into categories similar to subcommands - but I think this should be a follow-up PR :) . This one is big enough.

**Review by dimberman — APPROVED** [review]
LGTM

**Review by mik-laj — COMMENTED** [review]
Today it will definitely test it thoroughly.

**Comment by potiuk** [comment]
Fantastic @mik-laj !

File: hooks/build [hunk 1]
```diff
@@ -49,8 +49,9 @@ LAST_STEP_NAME=""
 STEP_STARTED="false"
 PYTHON_VERSION_FOR_DEFAULT_IMAGE=3.6
 VERBOSE=${VERBOSE:="false"}
+VERBOSE_COMMANDS=${VERBOSE_COMMANDS:="false"}
 
-if [[ ${VERBOSE} == "true" ]]; then
+if [[ ${VERBOSE_COMMANDS:="false"} == "true" ]]; then
```
Comment (ashb) [inline hooks/build]:
```suggestion
if [[ ${VERBOSE_COMMANDS} == "true" ]]; then
```

No default needed, we set it two lines up

File: scripts/ci/pre_commit_breeze_cmd_line.sh [hunk 2]
```diff
@@ -29,9 +28,23 @@ echo "
 .. code-block:: text
 " >"${TMP_FILE}"
 
-export SEPARATOR_WIDTH=80
-export AIRFLOW_CI_SILENT="true"
-./breeze --help | sed 's/^/  /' | sed 's/ *$//' >>"${TMP_FILE}"
+export MAX_SCREEN_WIDTH=100
```
Comment (ashb) [inline scripts/ci/pre_commit_breeze_cmd_line.sh:31]: 110 as that is the line length we have flake8 configured to?

**Review by ashb — APPROVED** [review]
Approved, looks good, but please look at the few small comments I added.

File: scripts/ci/pre_commit_breeze_cmd_line.sh (hunk 2 above)
Comment (potiuk) [reply scripts/ci/pre_commit_breeze_cmd_line.sh:31]: That too. But it's more an aesthetic thing :). The output looks nicer with it.

**Review by kaxil — APPROVED** [review]

**Review by mik-laj — APPROVED** [review]

**Pull request merged**